pkg/network/controllers/vm_test.go

```diff
@@ -244,6 +246,45 @@ var _ = Describe("VM Network Controller", func() {
 		Expect(iface.State).NotTo(Equal(v1.InterfaceStateAbsent))
 	})
 
+	DescribeTable("sync updates link state correctly", func(state v1.InterfaceState) {
```

Comment:
AFAIU, the VM and VMI both start with the same link state and transitions between states are not exercised.